.vsts-ci/phase.yml

```diff
@@ -60,6 +60,15 @@ jobs:
       condition: succeededOrFailed()
       continueOnError: ${{ parameters.continueonerror }}
 
+  - powershell: |
```

Comment:
To be completely honest, to this day, Azure DevOps has never made sense to me.  These config files just ramble on, and its so confusing.  I really wish that powershell could use a different CI.  Just saying.

Reply:
It's very flexible...
I love it.

Reply:
And it's Microsoft... So, I think you know the likelihood of us moving to something else.

Reply:
I can kinda see what you mean, but I just feel like all these different aspects could be improved a ton.  The docs also really don't help either.

Reply:
The language, probably could be a bit simpler and accomplish the same thing, but they had an existing GUI based system that they tried to fit into a YAML which somewhat limited their options.

Completely agree about the docs.

Reply:
Ouch, that's an awful port, GUI to YAML 👀 

Also, maybe they should just think about put the docs in here:
![](https://user-images.githubusercontent.com/34555510/67527633-783f2c00-f685-11e9-98f7-733f8322b4e7.jpeg)

Reply:
They at least need a major restructuring.

Reply:
Agreed